inst/rmarkdown/templates/bioinformatics/resources/template.tex

```diff
@@ -49,7 +49,7 @@
 % \usepackage[colorlinks=true, allcolors=blue]{hyperref}
 
 \access{Advance Access Publication Date: $access_date.day$ $access_date.month$ $access_date.year$}
-\appnotes{Manuscript Category}
+\appnotes{$manuscript_type$}
```

Comment:
I am wondering if we should maybe set a default Valueso that old Rmd file using previous version would still continue to build without this value set. 

Currently, it will be empty if no `manuscript_type` is set in the YAML header. 

Something like this as the `$subject_section$` was used incorrectly before. 

```suggestion
\appnotes{$if(manuscript_type)$$manuscript_type$$else$$subject_section$$endif$}
```

What do you think ?

Reply:
Agreed - wouldn't want to break something for someone else.